spiketoolkit/sortingcomponents/detection.py

```diff
@@ -7,29 +6,33 @@
 import numpy as np
 
 
-def detect_spikes(recording, channel_ids=None, detect_threshold=5, n_pad_ms=2, upsample=1, detect_sign=-1,
-                  min_diff_samples=5, align=True, start_frame=None, end_frame=None, n_jobs=1,
+def detect_spikes(recording, channel_ids=None, detect_threshold=5, detect_sign=-1,
+                  n_shifts=2, n_snippets_for_threshold=10, snippet_size_sec=1,
+                  start_frame=None, end_frame=None, n_jobs=1,
                   chunk_size=None, chunk_mb=500, verbose=False):
     '''
-    Detects spikes per channel.
+    Detects spikes per channel. Spikes are detected as threshold crossings and the threshold is in terms of the median
```

Comment:
No this is not threhold crossing technic. This is local extremam technic.
threshold was what you were doing.